Sources/Controllers/MyPlaces/OAOsmEditsListViewController.mm

```diff
@@ -62,13 +62,16 @@ @implementation OAOsmEditsListViewController
     BOOL _isSearchActive;
 }
 
-- (void)viewDidLoad {
+- (void)viewDidLoad 
+{
     [super viewDidLoad];
     _poiHelper = [OAPOIHelper sharedInstance];
     [self applySafeAreaMargins];
     _tableView.delegate = self;
     _tableView.dataSource = self;
     
+    self.segmentContainerView.backgroundColor = [UIColor.navBarBgColorPrimary colorWithAlphaComponent:1.0];
```

Comment:
colorWithAlphaComponent:1.0?

Reply:
Everything is correct! We discussed this problem when I view under the navbar because of the blur it turns out a different color